### ISSUE TITLE
Heading

### ISSUE THREAD

**Issue by izazw** [The-Backronym-Organization/project-4#28]
The illustration in h1 is meaningful text that is explaining the core functionality of the app. It might be a good Idea to put it in the text instead, or at least to add full text to the alt text (right now the "generator" word is missing.
